examples/js/exporters/OBJExporter.js

```diff
@@ -112,18 +114,21 @@ THREE.OBJExporter.prototype = {
 					}
 
 				}
-
+	
 				// faces
-
-
+	
 				for ( var i = 0, j = 1, l = faces.length; i < l; i ++, j += 3 ) {
 
 					var face = faces[ i ];
+					var indices = [];
```

Comment:
I moved it just outside of the faces loop. Not sure if it should be here or even higher, at the top of the `parse` method, for example.

(I rarely use for loops in my own code, instead opting for functional options, so I seldom think about where to hoist variables)